### DISCUSSION TITLE
Solve equation for x

### DISCUSSION THREAD

**Discussion by oondaasta** [josdejong/mathjs#3058] [Q&A] [answered]
I have the following expression:
`(3 - 6 * m) ^ 2 + (1 - 4 * m) ^ 2 + (1 - 2 * m) ^ 2 + 1 / 4 + (2 * m - 1) ^ 2 + (4 * m + -3 / 2) ^ 2 + (6 * m - 3) ^ 2`
and i create the derivative of it through: `derivative(expression, "m");`.
And i get the following derivative: `4 * (2 * m - 1) + 8 * (4 * m + -3 / 2) + 12 * (6 * m - 3) + -(12 * (3 - 6 * m)) - 8 * (1 - 4 * m) - 4 * (1 - 2 * m)`. 
Now i set to derivative equal to 0 and then solve for m. How could i do that?

**Comment by dvd101x** [comment, marked as answer]
I don't think there is a root finding algorithm yet in mathjs. You could import a function like this https://observablehq.com/@dvd101x/root-finding
